### PR TITLE
WIP Ana/fix delegation overview unit tests

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Maybe today is not my day. I don't get why, balances keeps being an empty Array :shrug: even passing it with setData

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f48f74a0d2e4bfbbd6aeca9--lunieio.netlify.app

File: app/src/components/common/TableBalances.vue [hunk 1]
```diff
@@ -13,6 +13,7 @@
           :balance="balance"
           :total-rewards-denom="totalRewardsDenom"
           :send="true"
+          :unstake="currentNetwork.network_type === 'polkadot'"
```
Comment (faboweb) [inline app/src/components/common/TableBalances.vue:16]: please revert. this button should appear in `Your Stake` only

File: app/tests/unit/specs/components/common/BalanceRow.spec.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ describe(`BalanceRow`, () => {
       },
     }
     wrapper.vm.$refs = $refs
-    wrapper.find(".table-cell.actions button").trigger("click")
+    wrapper.find("#send-button").trigger("click")
```
Comment (faboweb) [inline app/tests/unit/specs/components/common/BalanceRow.spec.js:71]: this can't be an id as the button might appear several times

**Comment by jbibla** [comment]
the unit test was failing because it was missing a prop. the changes in here aren't needed so i'll close and open a new PR with the fix.

**Comment by Bitcoinera** [comment]
there are fixes that are indeed needed. See commit `fix action buttons gone`. This needs to go in

**Comment by Bitcoinera** [comment]
image

**Pull request closed without merging**